tests/functional/comment/__init__.py

```diff
@@ -92,25 +95,57 @@ def setup_package():
 
     # Check the test project again.
     test_project_name = project_info['name'] + '_' + uuid.uuid4().hex
-
     ret = codechecker.check(codechecker_cfg,
                             test_project_name,
                             test_project_path)
     if ret:
         sys.exit(1)
-    print("Analyzing test project was succcessful.")
-
-    if pg_db_config:
-        print("Waiting for PotgreSQL to stop.")
-        codechecker.wait_for_postgres_shutdown(TEST_WORKSPACE)
+    print("Analyzing test project was successful.")
 
     codechecker_cfg['run_names'] = [test_project_name]
-
     test_config['codechecker_cfg'] = codechecker_cfg
-
     env.export_test_cfg(TEST_WORKSPACE, test_config)
 
 
+def enable_auth(workspace):
```

Comment:
Perhaps this method should be made generally available under `libtest` somewhere.